### PR TITLE
OLM 0.6.0 update

### PR DESCRIPTION
**Pull request by ecordell**
- Updates aggregated roles to list out verbs/resources explicitly
- Updates example prometheus object to include securityContext

### CODE REVIEW

**Comment by DanyC97** [comment]
OLM stands for ? would be nice to add a readme to this role so others can understand what its usage is ...

**Comment by ecordell** [comment]
@DanyC97 OLM is operator lifecycle manager: https://github.com/operator-framework/operator-lifecycle-manager

**Comment by DanyC97** [comment]
@ecordell thanks for the info. So could you pls create a readme in the current `olm` role and point to the info you just mentioned ?

**Comment by robszumski** [comment]
Not sure if you want to drop the `ocs` or not in this PR but we'll need change that before release.

File: roles/olm/files/20-aggregated-edit.clusterrole.yaml [hunk 1]
```diff
@@ -10,5 +10,5 @@ metadata:
     rbac.authorization.k8s.io/aggregate-to-edit: "true"
 rules:
 - apiGroups: ["operators.coreos.com"]
-  resources: ["*"]
-  verbs: ["*"]
\ No newline at end of file
+  resources: ["clusterserviceversions", "catalogsources", "installplans", "subscriptions"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```
Comment (enj) [inline roles/olm/files/20-aggregated-edit.clusterrole.yaml]: I have not reviewed the security implications of this, but `edit` should not include the verbs that are already in `view`.  View is automatically aggregated into edit.

**Pull request merged**